app/javascript/shared/components/emoji/emojisGroup.json

```diff
@@ -0,0 +1,9296 @@
+[
```

Comment:
@iamsivin Can you please share the reference link from where you have taken this? 

We can update it to v15.0 (https://unicode.org/emoji/charts/emoji-list.html) if you have not done that already.

Reply:
Taken it from [here](https://github.com/muan/unicode-emoji-json/blob/main/data-by-group.json) and edited the payload as we needed, and checked it with v15.0 (https://unicode.org/emoji/charts/emoji-list.html)